pkg/sql/sqlstats/insights/detector.go

```diff
@@ -170,3 +171,14 @@ func (d *latencyThresholdDetector) isSlow(s *Statement) bool {
 func isFailed(s *Statement) bool {
 	return s.Status == Statement_Failed
 }
+
+// ignoreStatement returns true if we don't want to analyze the statement.
+func ignoreStatement(s *Statement) bool {
```

Comment:
nit: can we name this function in a way that makes it clear it returns a bool? e.g. `isIgnoredStatement` or `shouldIgnoreStatement`